internal/resources/provisioner/provisioner_resource_test.go

```diff
@@ -0,0 +1,126 @@
+//go:build provisioner
+// +build provisioner
+
+/*
+Copyright © 2023 VMware, Inc. All Rights Reserved.
+SPDX-License-Identifier: MPL-2.0
+*/
+
+package provisioner
+
+import (
+	"fmt"
+	"os"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/acctest"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
+	"github.com/pkg/errors"
+
+	"github.com/vmware/terraform-provider-tanzu-mission-control/internal/authctx"
+	"github.com/vmware/terraform-provider-tanzu-mission-control/internal/client/proxy"
+	provisioner "github.com/vmware/terraform-provider-tanzu-mission-control/internal/models/provisioner"
+	testhelper "github.com/vmware/terraform-provider-tanzu-mission-control/internal/resources/testing"
+)
+
+func TestAcceptanceForProvisionerResource(t *testing.T) {
+	var provider = initTestProvider(t)
+
+	provisionerResourceName := fmt.Sprintf("%s.%s", ResourceName, resourceVar)
+	provisionerName := acctest.RandomWithPrefix("tf-prv-test")
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:          testhelper.TestPreCheck(t),
+		ProviderFactories: testhelper.GetTestProviderFactories(provider),
+		CheckDestroy:      nil,
+		Steps: []resource.TestStep{
+			{
+				Config: getTestProvisionerWithResourceConfigValue(provisionerName),
+				Check: resource.ComposeTestCheckFunc(
```

Comment:
Its better to add an update operation too in the test, you can create a ticket and add it later if there are time constraint

Reply:
Will take it up as part of the mock test PR or will take up later based on the time constraint

Reply:
Created jira - https://jira.eng.vmware.com/browse/TMC-49960